### ISSUE TITLE
Explain what Discord is for

### ISSUE THREAD

**Issue by Nautman** [digitalungdom-se/digitalungdom.se#136]
I think it would be neat to explain what the Discord is for and what you can do in our server. We could give examples such as:

* Asking for help
* Finding inspiration for new projects
* Sharing tech news
* etc

**Comment by Zigolox** [comment]
I agree, think we should have a big self-created discord widget and besides it some text explaining that we do most of our discussions there. Under that we can have your points as small sentences

**Comment by Zigolox** [comment]
I think this could be extra important during our Rays collaboration!

**Comment by Zigolox** [comment]
Works for now, but a more descriptive option would good in the future since discord is our main agora (forum). ;)

**Comment by Nautman** [comment]
I think I would very much appreciate if someone could help with inspiration for how it could look. That's the major problem for me, since I'm not very much in the mood start designing haha.

**Comment by Zigolox** [comment]
@charlesmaddock Do you have any ideas for a design?
